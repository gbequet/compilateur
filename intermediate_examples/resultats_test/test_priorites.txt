9
8
360
3
357
