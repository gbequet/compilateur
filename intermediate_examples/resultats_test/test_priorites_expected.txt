9
8
360
3
357
26
53
45
-35
v
v
v
f
v
f
v
f
